### ISSUE TITLE
QIIME 1.9.1

### ISSUE THREAD

**Issue by wasade** [biocore/conda-recipes#3]
...or would that be an insane request? It'd be nice if anything to lower Travis deployment times

**Comment by RNAer** [comment]
:+1:

**Comment by colinbrislawn** [comment]
Is that outside of the scope of this project?

I kind of like the idea of using this for individual, non-python programs like blast+ and HMMER. I don't really know the scope of this project, though.

**Comment by Jorge-C** [comment]
I personally think it'd be nice. If so, I can recover the recipe that we prepared for the 1.9.1RC, update it and issue a PR in the next few days.

**Comment by RNAer** [comment]
I would argue for both types of packages. @colinbrislawn (or others), what
are the motivations not to do that? I may miss those.

**Comment by colinbrislawn** [comment]
@RNAer Good point. Including more software would only make Travis-CI builds slower, or maybe not! #6 

I've been deploying qiime with conda for the last year now and it works great. If this project makes it easier to get new versions of qiime on binstar, let's do it.

**Comment by gregcaporaso** [comment]
I think this is a good idea, and probably makes the most sense for there to be a QIIME base install recipe, and then recipes for additional components as we want them (e.g., RDP Classifier would be a good one).

**Comment by ElDeveloper** [comment]
Just FYI, @bgruening posted about QIIME being available through bioconda https://github.com/biocore/qiime/issues/1759#issuecomment-191116405

**Comment by colinbrislawn** [comment]
Package is currently under development in bioconda:
https://github.com/bioconda/bioconda-recipes/tree/master/recipes/qiime/1.9.1 

PR to get OSX working:
https://github.com/bioconda/bioconda-recipes/pull/922

**Comment by bgruening** [comment]
OS-X builds are now available. Please test!
